components/topbar/src/application.styles.ts

```diff
@@ -22,4 +30,17 @@ export const useStyles = makeStyles({
   disabled: {
     opacity: 0.3,
   },
+  selectedApp: {
+    [`& .${iconRegularClassName}`]: {
+      display: "none",
+    },
+    [`& .${iconFilledClassName}`]: {
+      color: tokens.colorCompoundBrandForeground1,
+      display: "inline",
+    },
+  },
+  selectedAppLabel: {
+    ...typographyStyles.body1Strong,
+    color: tokens.colorNeutralForeground1,
+  },
```

Comment:
@markuslindmark can you add your approval so we can trace in the future that this was done with your blessing? 😄

Reply:
I mean, I know it was, but I'll forget in a month or so when someone asks "why is the app list like that".